presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestMultipleDistinctAggregationToMarkDistinct.java

```diff
@@ -29,12 +27,9 @@ public void testNoDistinct()
         tester().assertThat(new SingleDistinctAggregationToGroupBy())
                 .on(p -> p.aggregation(builder -> builder
                         .globalGrouping()
-                        .addAggregation(p.variable("output1"), expression("count(input1)"), ImmutableList.of(BIGINT))
-                        .addAggregation(p.variable("output2"), expression("count(input2)"), ImmutableList.of(BIGINT))
-                        .source(
-                                p.values(
-                                        p.variable("input1"),
-                                        p.variable("input2")))))
+                        .source(p.values(p.variable("input1"), p.variable("input2")))
```

Comment:
by convention we put source as the last argument and then indent the args so you can see the plan shape.